examples/custom-server-express/server.js

```diff
@@ -0,0 +1,37 @@
+const express = require('express')
+const next = require('next')
+const accepts = require('accepts')
+
+const app = next('.', { dev: true })
```

Comment:
how about adding a comment in the example stating `The first argument is also optional and . is default`